tests/test_data_processor.py

```diff
@@ -499,11 +555,58 @@ def test_max_altitude(self, data_processor):
             ),
         ],
     )
-    def test_calculate_rotations(self, csv_path, expected_value, n_packets):
+    def test_calculate_rotations_quaternions(self, csv_path, expected_value, n_packets):
         data_packets = load_data_packets(csv_path, n_packets)
         d = IMUDataProcessor()
         d.update(data_packets)
         rotations = d._rotated_accelerations
         assert len(rotations) == n_packets
 
         assert rotations[-1] == pytest.approx(expected_value)
```

Comment:
```suggestion
        assert rotations[-1] == pytest.approx(expected_value)
        assert rotations[0] == rotations[-1]
```
this checks that all the rotations set were the same for that list for the quaternion only case.